superbench/benchmarks/micro_benchmarks/cudnn_function.py

```diff
@@ -424,8 +425,8 @@ def _process_raw_result(self, cmd_idx, raw_output):
                     raw_data = raw_data.split(',')
                     raw_data.pop()
                     raw_data = [float(item) for item in raw_data]
-                    self._result.add_result(metric, sum(raw_data) / len(raw_data))
-                    self._result.add_raw_data(metric, raw_data)
+                    self._result.add_result(metric.lower() + '_time', statistics.mean(raw_data) * 1000)
```

Comment:
why `* 1000`?

Reply:
because I want to unify them to us, the unit is ms original for cudnn, while the unit of cublas is already us